core/fullHistory/historyProcessor.go

```diff
@@ -17,38 +18,38 @@ import (
 
 var log = logger.GetOrCreate("core/fullHistory")
 
-// HistoryProcessorArguments is the structure that store all components that are needed to a history processor
-type HistoryProcessorArguments struct {
+// HistoryRepositoryArguments is a structure that stores all components that are needed to a history processor
+type HistoryRepositoryArguments struct {
```

Comment:
Name of file remained unchanged - perhaps rename it?

Reply:
done.